integration/entropy/entropy_test.bats

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# The main purpose of this test is to print
+# the entropy level inside the container after
+# we installed the haveged package. We need to
+# verify that the entropy level is not < 1000
+# https://wiki.archlinux.org/index.php/Haveged
+
+source /etc/os-release || source /usr/lib/os-release
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+# Environment variables
+IMAGE="busybox"
+# This the minimum entropy level produced
+# by haveged is 1000 see https://wiki.archlinux.org/index.php/Haveged
+# Less than 1000 could potentially slow down cryptographic
+# applications see https://www.suse.com/support/kb/doc/?id=7011351
```

Comment:
@sboeuf , here I am putting an explanation about why this number and with their respective reference, thanks

Reply:
Oh sorry I missed that...